src/main/java/com/github/splendor_mobile_game/websocket/handlers/reactions/GetTokens.java

```diff
@@ -1,74 +1,101 @@
 package com.github.splendor_mobile_game.websocket.handlers.reactions;
 
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.TooManyListenersException;
+import java.util.UUID;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
 import com.github.splendor_mobile_game.database.Database;
+import com.github.splendor_mobile_game.game.enums.Color;
+import com.github.splendor_mobile_game.game.enums.TokenType;
+import com.github.splendor_mobile_game.game.model.Room;
+import com.github.splendor_mobile_game.game.model.User;
+import com.github.splendor_mobile_game.websocket.communication.ServerMessage;
 import com.github.splendor_mobile_game.websocket.communication.UserMessage;
+import com.github.splendor_mobile_game.websocket.handlers.DataClass;
 import com.github.splendor_mobile_game.websocket.handlers.Messenger;
 import com.github.splendor_mobile_game.websocket.handlers.Reaction;
 import com.github.splendor_mobile_game.websocket.handlers.ReactionName;
+import com.github.splendor_mobile_game.websocket.handlers.ServerMessageType;
+import com.github.splendor_mobile_game.websocket.handlers.exceptions.GameNotStartedException;
+import com.github.splendor_mobile_game.websocket.handlers.exceptions.InvalidUUIDException;
+import com.github.splendor_mobile_game.websocket.handlers.exceptions.NotThisUserTurnException;
+import com.github.splendor_mobile_game.websocket.handlers.exceptions.RoomDoesntExistException;
+import com.github.splendor_mobile_game.websocket.handlers.exceptions.TooManyReturnedTokensException;
+import com.github.splendor_mobile_game.websocket.handlers.exceptions.TooManyTokensException;
+import com.github.splendor_mobile_game.websocket.handlers.exceptions.UserNotAMemberException;
+import com.github.splendor_mobile_game.websocket.handlers.exceptions.UserNotFoundException;
+import com.github.splendor_mobile_game.websocket.handlers.exceptions.WrongTokenChoiceException;
+import com.github.splendor_mobile_game.websocket.response.ErrorResponse;
+import com.github.splendor_mobile_game.websocket.response.Result;
+import com.github.splendor_mobile_game.websocket.utils.Log;
 
 /**
  * Players send this request if now is their turn and they want to get tokens from the table.
  * In reaction server sends to all players message of type `GET_TOKENS_ANNOUNCEMENT` announcing that this has happend.
  * 
  * Example of user request
  * {
- *      "messageContextId": "02442d1b-2095-4aaa-9db1-0dae99d88e03",
+ *      "contextId": "02442d1b-2095-4aaa-9db1-0dae99d88e03",
  *      "type": "GET_TOKENS",
  *      "data": {
- *          "tokensChange": {
- *              "red": 1,
- *              "blue": 1,
- *              "green": 1,
- *              "white": 0,
- *              "black": 0
+ *          "tokensChangeDTO": {
+ *              "RUBY": 1,
+ *              "SAPPHIRE": 1,
+ *              "EMERALD": 1,
+ *              "DIAMOND": 0,
+ *              "ONYX": 0
```

Comment:
You need to change those names to lowercase. Raw json message provided with UPPERCASE doesn't work.